src/backend/InvenTree/InvenTree/settings.py

```diff
@@ -1560,3 +1561,76 @@
 
 if SITE_URL and not TESTING:  # pragma: no cover
     SPECTACULAR_SETTINGS['SERVERS'] = [{'url': SITE_URL}]
+
+# Storage backends
+STORAGE_TARGET = get_setting(
+    'INVENTREE_STORAGE_TARGET', 'storage.target', 'local', typecast=str
```

Comment:
We should validate that the provided `INVENTREE_STORAGE_TARGET` variable is one of the valid options we actually support.